etc/checkstyle-custom_checks.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
+
+<!--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+<module name="Checker">
+    <module name="SuppressWarningsFilter"/>
+
+    <module name="TreeWalker">
+        <module name="IllegalToken">
+            <property name="tokens" value="LABELED_STAT, LITERAL_ASSERT"/>
```

Comment:
Included `LABELED_STAT` as its the default for the check and seemed wise to preserve.